js/app.jsx

```diff
@@ -517,6 +537,8 @@ var Job = React.createClass({
       ['State', state],
       ['Input', inputs],
       ['Output', cleanJobPath(job.conf.output)],
+      ['Pipe', pipe],
+      ['Step', step],
```

Comment:
@alyssafrazee So the "Scalding steps" for that task is:
```
com.stripe.zoolander.parquet.ParquetJob$class.load(ParquetJob.scala:191),
com.stripe.zoolander.parquet.ParquetJob$TypedPipeExtensions.save(ParquetJob.scala:201),
com.stripe.zoolander.transactionfraud.JoinModelProbShieldData.<init>(JoinModelProbShieldData.scala:104),
com.stripe.zoolander.transactionfraud.JoinModelProbShieldData.<init>(JoinModelProbShieldData.scala:106)
```
and the "Scalding pipes" are
```
com.stripe.zoolander.transactionfraud.JoinModelProbShieldData.<init>(JoinModelProbShieldData.scala:106)
com.stripe.zoolander.parquet.ParquetJob$TypedPipeExtensions.save(ParquetJob.scala:201)
```